wiki/Modding/en.md

```diff
@@ -1,98 +1,237 @@
+## Getting Started
 
-Next, go into the [Editor](Editor "wikilink") and select the map which you want to mod. Then, start up a reply to the [Beatmap's](Beatmap "wikilink") thread.
+To get started, go to the [Pending](https://osu.ppy.sh/forum/6) or [Work In Progress/Help](https://osu.ppy.sh/forum/10) subforums.
+From here, you can either randomly choose beatmaps from these subforums to mod or open a modding queue in the [Modding Queue](https://osu.ppy.sh/forum/60) subforum and accept requests.
+If you are opening a queue for the first time, it is a good idea to see how modding queues work by looking at some of the existing threads in the Modding Queue subforum.
 
-Choose any of the difficulties to begin your mod. It is recommended to start with the easiest or hardest difficulty and make your way up or down the different difficulties.
+Open the beatmap's forum thread by clicking on `Web` then `This Beatmap's Thread`.
+From here, click the `Reply` button on the bottom-left to start your mod post.
+In this post, you should add a heading to denote the difficulty(ies) you are modding.
 
-Start by playing this difficulty in either Play mode or [test mode](Glossary#Test_Mode "wikilink"). Play-testing is an important part of modding as it allows the modder to understand whether or not certain [patterns](Mapping_Techniques "wikilink") play well. Take note of anything that felt odd.
+Next, open the song in the osu!client.
+You can choose any of the difficulties to start a mod but it is recommended to start with the easiest difficulty and continue with harder difficulties or vise versa.
 
-After play-testing, watch the song in the editor and pause frequently to catch any errors. In this stage of modding, it is most often that modders find more intricate problems such as combo errors, hitsound inconsistencies, and areas where hit circles or sliders are missing. When you find a hit circle or a slider that needs attention, select it. Now, go to the "Edit" tab and select "Copy" at the top (or Ctrl+C), and then to your post you started up, and paste (Ctrl+V) what you just copied there. Then, follow up with either what you think is wrong (Ex. "I would change this spacing because the music does not suggest a [jump](jumps "wikilink") here), or simply that it doesn't seem right and you can't put your finger on it. Always look for things that you didn't notice while playing.
+Start by playing the selected difficulty in test mode.
```

Comment:
Alrighty, changing to say `Before going to edit mode, play the difficulties you want to mod in solo mode.`